aten/src/ATen/native/cuda/Dropout.cu

```diff
@@ -18,6 +21,77 @@ namespace {
 // for all members of float4 to be consumed UNROLL has to be 4. Don't change!
```

Comment:
@slayton58 Per this note, what if VEC != 4? I guess maybe this note should be changed to `UNROLL has to be <=4`?

Reply:
`VEC <= 4` would be the correct change here, yes